daft/daft.pyi

```diff
@@ -994,6 +991,12 @@ class PyExpr:
     def partitioning_iceberg_truncate(self, w: int) -> PyExpr: ...
     def json_query(self, query: str) -> PyExpr: ...
 
+    ###
+    # Helper methods required by optimizer:
+    # These should be removed from the Python API for Expressions when logical plans and optimizer are migrated to Rust
+    ###
+    def _input_mapping(self) -> builtins.str | None: ...
+
```

Comment:
The only place that this is still (transitively) used in Python land is here: https://github.com/Eventual-Inc/Daft/blob/6db19a6b80d2962faea652670a7b396a0f9f700f/daft/execution/execution_step.py#L470

Looking forward to removing this in the near-term!